pkg/solana/logpoller/log_poller.go

```diff
@@ -316,10 +316,10 @@ func (lp *Service) getLastProcessedSlot(ctx context.Context) (lastProcessed int6
 		return 0, err
 	}
 	if lookbackSlot > lastProcessed {
-		lp.lggr.Infow("last processed slot is older than lookback window, skipping ahead", "lastProcessed", lastProcessed, "lookbackSlot", lookbackSlot)
+		lp.lggr.Infow("last processed slot is older than lookback window, skipping ahead to lookback slot", "last_processed", lastProcessed, "lookback_slot", lookbackSlot)
 		return lookbackSlot, nil
 	}
-	lp.lggr.Infow("last processed slot is still within lookback window, resuming at last processed slot", "lastProcessed", lastProcessed, "lookbackSlot", lookbackSlot)
```

Comment:
Originally, this commit changed all the Infof's to Infow's, but looks like that just went into another commit I rebased on top of this morning so... not much is different.